Src/ReleaseHistory.md

```diff
@@ -8,6 +8,7 @@
 - Update `sarif-sdk` submodule to commit [7ddf923a4652a333f3356e9db4c5742b78a22c96](https://github.com/microsoft/sarif-sdk/commit/7ddf923a4652a333f3356e9db4c5742b78a22c96), which includes threading fixes in driver framework and fixes to ensure snipet population.
 - Update `spam` executable and dotnet library name to Sarif.PatternMatcher.Cli. 
 - Update `Microsoft.Security.Utilities` to [v1.4.0](https://github.com/microsoft/security-utilities/releases/tag/v1.4.0). [#662](https://github.com/microsoft/sarif-pattern-matcher/pull/662)
+- Rename IdentifiableNpmAuthorToken to NpmAuthorIdentifiableToken [#683](https://github.com/microsoft/sarif-pattern-matcher/pull/683)
```

Comment:
>IdentifiableNpmAuthorToken to NpmAuthorIdentifiableToken [](http://example.com/codeflow?start=9&length=56)

Almost there! Please include the RuleId (SEC101/050) and enclose the rule names in backticks.

Something like:
```
Rename `SEC101/050.IdentifiableNpmAuthorToken` to `SEC101/050.NpmAuthorIdentifiableToken` 
```